plugins/VersionUpgrade/VersionUpgrade462to47/VersionUpgrade462to47.py

```diff
@@ -27,6 +36,11 @@ def upgradePreferences(self, serialized: str, filename: str) -> Tuple[List[str],
 
         # Update version number.
         parser["metadata"]["setting_version"] = "15"
+        
+        # Remove deleted settings from the visible settings list.
```

Comment:
You should also remove them from the profiles.

Reply:
Good catch, thx.